jetty-server/src/main/config/etc/jetty-http.xml

```diff
@@ -42,6 +42,7 @@
         <Get name="SelectorManager">
           <Set name="connectTimeout"><Property name="jetty.http.connectTimeout" default="15000"/></Set>
         </Get>
+        <Set name="name">http</Set>
```

Comment:
Default `jetty-http.xml` should not have a name set.
Perhaps it could have an optional `<Property>` but not a specific name set.

Reply:
good idea :+1:

Reply:
`<Set name="name"><Property name="jetty.http.connectorName" default=""/></Set>`

how about this?

Reply:
I don't particularly mind having default names for our connectors.  This one could be "http", but note that "https" does not work (see below).
We should make sure the setter treats "" the same as null.